integration_tests/mockApis/lostBed.ts

```diff
@@ -20,6 +20,19 @@ export default {
       },
     }),
 
+  stubLostBed: (args: { premisesId: string; lostBed: LostBed }): SuperAgentRequest =>
+    stubFor({
+      request: {
+        method: 'GET',
+        url: `/premises/${args.premisesId}/lost-beds/${args.lostBed.id}`,
```

Comment:
I'm not doing it as "change-all-at-once" things, but in TA, I'm starting to use static-paths here, rather than plain strings

Reply:
It's a good plan. I was just lazy and copied it from elsewhere 😁